sormas-ui/src/main/java/de/symeda/sormas/ui/utils/SormasGrid.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2022 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.ui.utils;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.vaadin.v7.ui.Grid;
+
+import de.symeda.sormas.api.FacadeProvider;
+import de.symeda.sormas.api.utils.fieldvisibility.FieldVisibilityCheckers;
+
+public class SormasGrid extends Grid {
```

Comment:
If I remember correctly, @MartinWahnschaffe is not a huge fan of naming classes (or any elements, for that matter) according to the project they are initially implemented in (e.g. because it makes it harder to reuse these elements for other projects). I'd recommend trying to come up with a way that describes what the class is used for.